packages/showcases/react/stories/components/VtmnDivider/VtmnDivider.stories.tsx

```diff
@@ -0,0 +1,162 @@
+    <div>
+      Lorem ipsum dolor sit amet consectetur adipisicing elit.
+      <VtmnDivider className="vtmn-my-4" />
+      Lorem ipsum dolor sit amet consectetur adipisicing elit.
+    </div>
+
+    <div>
+      <VtmnDivider className="vtmn-my-4">Default</VtmnDivider>
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+      <VtmnDivider className="vtmn-my-4" text="start">
+        Start
+      </VtmnDivider>
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+      <VtmnDivider className="vtmn-my-4">Center</VtmnDivider>
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+      <VtmnDivider className="vtmn-my-4" text="end">
+        End
+      </VtmnDivider>
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+    </div>
+  </div>
+);
+
+export const Horizontal = HorizontalTemplate.bind({});
+
+/**
+ * Vertical Template
+ */
+
+const VerticalTemplate: Story = (args) => (
+  <div className="vtmn-flex vtmn-flex-col vtmn-items-center vtmn-space-y-8 vtmn-text-center vtmn-typo_text-2">
+    <div className="vtmn-flex vtmn-w-1/2">
+      <p>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Vitae,
+        adipisci? Aspernatur laboriosam ducimus sapiente incidunt?
+      </p>
+      <VtmnDivider className="vtmn-mx-4" orientation="vertical" />
+      <p>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Vitae,
+        adipisci? Aspernatur laboriosam ducimus sapiente incidunt?
+      </p>
+    </div>
+
+    <div className="vtmn-flex vtmn-w-8 vtmn-justify-center">
+      <VtmnDivider className="vtmn-mx-4" orientation="vertical">
+        Default
+      </VtmnDivider>
+      <p>
+        Lorem ipsum dolor sit amet consectetur, adipisicing elit. Non
+        consequuntur inventore aperiam hic voluptates asperiores minima cumque
+        quos ab cupiditate.
+      </p>
+
+      <VtmnDivider className="vtmn-mx-4" orientation="vertical" text="start">
+        Start
+      </VtmnDivider>
+      <p>
+        Lorem ipsum dolor sit amet consectetur, adipisicing elit. Non
+        consequuntur inventore aperiam hic voluptates asperiores minima cumque
+        quos ab cupiditate.
+      </p>
+
+      <VtmnDivider className="vtmn-mx-4" orientation="vertical" text="center">
+        Center
+      </VtmnDivider>
+      <p>
+        Lorem ipsum, dolor sit amet consectetur adipisicing elit. Officiis
+        officia dolorem, voluptatibus obcaecati explicabo cum doloremque dolorum
+        nulla ipsa consequuntur?
+      </p>
+
+      <VtmnDivider className="vtmn-mx-4" orientation="vertical" text="end">
+        End
+      </VtmnDivider>
+      <p>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Magni quidem
+        architecto cum assumenda ad possimus modi harum voluptas earum in?
+      </p>
+    </div>
+  </div>
+);
+
+export const Vertical = VerticalTemplate.bind({});
```

Comment:
We need to use the same behavior as other React storybook stories like `VtmnButton`, `VtmnToggle` etc. :)

```suggestion
import React from 'react';
import { VtmnButton } from '@vtmn/react';
import { Meta, Story } from '@storybook/react';
import {
  argTypes,
  parameters,
} from '@vtmn/showcase-core/csf/components/VtmnDivider.csf';

export default {
  title: 'Components/VtmnDivider',
  component: VtmnDivider,
  argTypes: {
    ...argTypes,
    children: {
      type: { name: 'string', required: false },
      description: 'The content to render inside the component',
      defaultValue: 'Button',
      control: {
        type: 'text',
      },
    },
  },
  parameters,
} as Meta;

const Template: Story = (args) => <VtmnDivider {...args} />;

export const Overview = Template.bind({});
```

Reply:
Thanks for the review @lauthieb  ! I may have a question about the storybook part, in fact the divider doesn't render as it should by calling <VtmnDivider {...args} /> without a container that wraps it, this is mainly why I've done examples with text, fleboxes and grids ->

Without container :
![Screenshot 2021-12-31 at 18-05-28 Components VtmnDivider - Overview ⋅ Storybook](https://user-images.githubusercontent.com/20931885/147833571-8bf36ae7-d448-4713-a5fd-0e777cff05de.png)

With container (200x200 pixels) wrapping <VtmnDivider {...args} />
![Screenshot 2021-12-31 at 18-06-06 Components VtmnDivider - Overview ⋅ Storybook](https://user-images.githubusercontent.com/20931885/147833597-3e8909af-2fca-4118-a609-5126ef877409.png)

There are also some ( strange ? ) behaviors that I forgot to mention, for example when there isn't enough content I think the divider works the other way around with the orientation set to vertical  : 

![Screenshot 2021-12-31 at 18-13-50 Components VtmnDivider - Overview ⋅ Storybook](https://user-images.githubusercontent.com/20931885/147833817-217381b3-e8fe-4e1b-87a9-5a44ed61fe9d.png)

Shouldn't textPosition=end place text at the bottom with most of the line above the label and the rest under the label like here ?

![image](https://user-images.githubusercontent.com/20931885/147834261-afa0b3f8-958b-4b8b-acaa-44814379aaea.png)


These weird placements do not occur with enough text. From what I understood it occurs when the height of the divider isn't tall enough.